modules/veteran/app/workers/veteran/base_reloader.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module Veteran
+  class BaseReloader
+    include Sidekiq::Worker
+    BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
+
+    private
+
+    def find_or_initialize(hash)
+      rep = Veteran::Service::Representative.find_or_initialize_by(representative_id: hash['Registration Num'],
+                                                                   first_name: hash['First Name'],
+                                                                   last_name: hash['Last Name'])
+      rep.poa_codes ||= []
+      rep.user_types ||= []
+      rep.poa_codes << hash['POA Code'].gsub!(/\W/, '')
+      rep.phone = hash['Phone']
+      rep
+    end
+
+    def fetch_data(action)
```

Comment:
you may have already spent time cleaning this up but wondering if it could be tidied up a bit.. is excel -> html -> csv -> hash the best approach or the first that worked?

Reply:
The data actually comes in as XML so it never really enters the excell, html or csv formats, hash in my opinion is the cleanest implementation to get it into before persisting to the DB.

Reply:
Can't help but agree here, transforming the data through even 3 different formats feels weird, Seems like you could just go XML -> hashes.

I know you say that it does enter html or csv formats, but is that not what the block that starts on 26 doing? Then you map over the contents of the CSV calling `to_h` to get your array of hashes

Reply:
So no I took another look and I was wrong, It comes in as HTML and I can convert it straight to Hash for persistence but it's never in xml or excel, the CSV parsing was just the quickest way to do it at the time but I'll modify it to go straight from HTML to Hash in this case.

Reply:
Done